test/unit/ngsiv2/examples/contextRequests/updateContextProcessTimestamp.json

```diff
@@ -0,0 +1,17 @@
+{
+  "state":{
+    "type": "Boolean",
+    "value": true,
+    "metadata": {
+      "TimeInstant":{
+        "type": "DateTime",
+        "value": "2016-05-30T16:25:22.304Z"
+      }
+    }
+  },
+  "TimeInstant":
+  {
+    "type": "DateTime",
+    "value": "2016-05-30T16:25:22.304Z"
+  }
+}
```

Comment:
newline expected here

Reply:
Fixed in 48e3530f18fee235e67e9cdc53fdf992b72e69a7